scalac-native/README.md

```diff
@@ -2,40 +2,34 @@
 
 This demo demonstrates how to build a native image of the Scala compiler.
```

Comment:
a native image of the Scala compiler. > a native executable version of the Scala compiler.